packages/components/src/components/tag/tag.stories.ts

```diff
@@ -1,21 +1,24 @@
 import '../../solid-components';
 import { html } from 'lit';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
-import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-tag');
 const { generateTemplate } = storybookTemplate('sd-tag');
 const { overrideArgs } = storybookHelpers('sd-tag');
 
+/**
+ *
+ *  Tags are used to categorize or label content. They can be used to filter or search for content.
```

Comment:
*  Used to categorize or label content. Can also be used to filter or search for content.

Reply:
Updated